apply-patches.sh deletes this file from the source directory.

--- otp_src_R16B02.old/lib/tools/emacs/Makefile	2012-04-04
+++ otp_src_R16B02/lib/tools/emacs/Makefile	2012-04-04 15:55:16.978957307 +0100
@@ -51,7 +51,7 @@

 ELC_FILES = $(EMACS_FILES:%=%.elc)

-TEST_FILES = test.erl.indented test.erl.orig
+TEST_FILES = test.erl.indented

 # ----------------------------------------------------
 # Targets
